src/components/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import COLORS from '../styles/colors';
+
+export const Button = styled.button<{ $secondary?: boolean }>`
+  /* Adapt the colors based on the secondary prop */
+  color: ${props => (props.$secondary ? 'black' : 'white')};
+  background: ${props => (props.$secondary ? 'white' : COLORS.ijpBlue)};
+
+  appearance: none;
+  align-self: flex-end;
+  padding: 10px 20px;
+  border-radius: 5px;
+  border: 2px solid ${COLORS.ijpBlue};
+  &:hover {
+    background: ${props =>
+      props.$secondary ? '#f2f2f2' : COLORS.ijpBlueClicked};
+    border-color: ${COLORS.ijpBlueClicked};
+  }
+`;
+
+export const ErrorButton = styled(Button)<{ $secondary?: boolean }>`
+  /* Adapt the colors based on the secondary prop for error buttons */
+  background: ${props => (props.$secondary ? 'white' : COLORS.errorColor)};
+  border: 2px solid ${COLORS.errorColor};
+  &:hover {
+    background: ${props =>
+      props.$secondary ? '#f2f2f2' : COLORS.errorColorClicked};
```

Comment:
do we want to use a color from COLORS, or add this color (`'#f2f2f2'`) to COLORS?

Reply:
oh I already added it as a color....

Reply:
Updating this! Also, I'll update the COLORS.tsx file so that we don't have more than one variable for the same color. I think it could be good to stick with the names David gave the colors on Figma.